tests/DbaaS/pages/dbaasActionsPage.js

```diff
@@ -72,10 +72,11 @@ module.exports = {
     }
   },
 
-  async deleteXtraDBCluster(dbClusterName, k8sClusterName) {
+  async deleteXtraDBCluster(dbClusterName, k8sClusterName, deleteCompleted = true) {
```

Comment:
Please use the CamelCase for the functions names. See: https://github.com/percona/pmm-ui-tests/blob/main/CONTRIBUTING.md#naming-conventions

Reply:
no related to my changes, will fix all function names later in refactoring task